### PR TITLE
The method base_run_model::update_progress_for_index() does not seem to be used

### PR DESCRIPTION
**Pull request by BjarneHerland**

### CODE REVIEW

**Comment by BjarneHerland** [comment]
Jenkins: test ert please

**Comment by BjarneHerland** [comment]
Jenkins: test this please

File: ert_shared/models/base_run_model.py [hunk 1]
```diff
@@ -190,55 +190,6 @@ def get_runtime(self) -> Union[int, float]:
...
-
-            if loaded:
-                jobs = loaded.jobs
-
-        if timed_out:
-            for job in jobs:
-                if job.status != "Success":
-                    job.error = "The run is cancelled due to reaching MAX_RUNTIME"
-                    job.status = "Failure"
-        self.realization_progress[iteration][run_arg.iens] = jobs, status
```
Comment (jondequinor) [inline ert_shared/models/base_run_model.py:240]: Do you still need `self.realization_progress: Dict[int, int] = {}` then?

File: ert_shared/models/base_run_model.py [hunk 2]
```diff
@@ -190,55 +190,6 @@ def get_runtime(self) -> Union[int, float]:
...
-            ]
-            and not timed_out
-        ):
-            return
-
-        fms = self.realization_progress[iteration].get(run_arg.iens, None)
-        jobs = fms[0] if fms else None
-
-        # Don't load from file if you are finished
-        if not fms or not BaseRunModel.is_forward_model_finished(fms[0]):
```
Comment (jondequinor) [inline ert_shared/models/base_run_model.py:224]: Might also make `is_forward_model_finished` superfluous.

File: ert_shared/models/base_run_model.py [hunk 3]
```diff
@@ -190,55 +190,6 @@ def get_runtime(self) -> Union[int, float]:
...
-            # will throw if not yet submitted (is in a limbo state)
-            queue_index = run_arg.getQueueIndex()
-        except (ValueError, AttributeError):
-            return
-
-        status = None
-        timed_out = False
-        if self._job_queue:
-            status = self._job_queue.getJobStatus(queue_index)
-            timed_out = self._job_queue.did_job_time_out(queue_index)
```
[the remark this answers is not in the document]
Comment (BjarneHerland) [reply ert_shared/models/base_run_model.py:204]: **sigh**  yes, you are right...

File: ert_shared/models/base_run_model.py (hunk 2 above)
Comment (BjarneHerland) [reply ert_shared/models/base_run_model.py:224]: It's used here
https://github.com/equinor/ert/blob/7d0254f5de6b6d4e1df4774c51962f07ab62474e/tests/ert_tests/models/test_base_run_model.py#L32-L33
With my (limited) insight into the details of the queue-mechanism, this seems like a reasonable test. No...?

File: ert_shared/models/base_run_model.py (hunk 1 above)
Comment (BjarneHerland) [reply ert_shared/models/base_run_model.py:240]: Probably not

File: ert_shared/models/base_run_model.py (hunk 2 above)
Comment (jondequinor) [reply ert_shared/models/base_run_model.py:224]: Yes, you are right!

**Comment by jondequinor** [comment]
did you decide to keep `did_job_time_out`? I see the conversation as resolved, but it is still there.

**Comment by BjarneHerland** [comment]
> did you decide to keep `did_job_time_out`? I see the conversation as resolved, but it is still there.

Sorry - mea culpa! Pushed new version.

**Comment by BjarneHerland** [comment]
Jenkins test this please

**Review by jondequinor — APPROVED** [review]
LGTM! Thanks

**Comment by BjarneHerland** [comment]
Jenkins test this please

**Pull request merged**